components/ingest-service/backend/elastic/nodes.go

```diff
@@ -275,7 +273,6 @@ func (es *Backend) deleteNodeByDocID(ctx context.Context, docIds []string) (int,
 	for _, docID := range docIds {
 		updateReq := elastic.NewBulkUpdateRequest().
 			Index(mappings.NodeState.Alias).
-			Type(mappings.NodeState.Type).
```

Comment:
You removed the type which is good, but do we need to apply that type as a filter?

Reply:
it is not needed in opensearch